Configuration/Settings.yaml

```diff
@@ -34,6 +34,15 @@ Neos:
           'Neos.Neos.UI:HostOnlyStyles':
             resource: '${"resource://" + Neos.Ui.StaticResources.compiledResourcePackage() + "/Public/Styles/HostOnlyStyles.css"}'
             position: 'start 900'
+          'WorkSans:200':
```

Comment:
@Inkdpixels hey nice to see you here again!
I'm sure it was just a mistake, we should use the definition from the brands package ofc.